docs/content/installation/csi_ug_requirements.md

```diff
@@ -50,20 +50,47 @@ Perform these steps for each worker node in Kubernetes cluster to prepare your e
    Apply the yaml file.
 
    `oc apply -f 99-ibm-attach.yaml`
+
+3. Configure storage system connectivity.
+
+    1.  Define the host of each Kubernetes node on the relevant storage systems with the valid WWPN (for Fibre Channel) or IQN (for iSCSI) of the node.
+
+    2.  For Fibre Channel, configure the relevant zoning from the storage to the host.
     
-3. If needed, enable support for volume snapshots (FlashCopy® function) on your Kubernetes cluster.
+4. (Optional) If planning on using volume snapshots (FlashCopy® function), enable support on your Kubernetes cluster.
 
    For more information and instructions, see the Kubernetes blog post, [Kubernetes 1.20: Kubernetes Volume Snapshot Moves to GA](https://kubernetes.io/blog/2020/12/10/kubernetes-1.20-volume-snapshot-moves-to-ga/).
 
    Install both the Snapshot CRDs and the Common Snapshot Controller once per cluster.
 
    The instructions and relevant yaml files to enable volume snapshots can be found at: [https://github.com/kubernetes-csi/external-snapshotter#usage](https://github.com/kubernetes-csi/external-snapshotter#usage)
 
-4. Configure storage system connectivity.
+5. (Optional) If planning on using volume replication (mirroring), enable support on your Kubernetes cluster.
```

Comment:
I'm not sure, but by comparing to the snapshots docs, there might be a `RemoteCopy® function` missing here

Reply:
@oriyarde  Who can provide me with this information if (and what) is missing?

Reply:
how did we know to include `FlashCopy® function` for snapshots?

Reply:
@oriyarde  are you talking about the name usage? or a specific command/step?
If you're talking about the name - I know because I've worked quite a bit with the SV team and am familiar with the terminology.
Here are the [available terms](https://www.ibm.com/docs/en/flashsystem-9x00/8.4.x?topic=overview-copy-services-functions) for SV content for the various copy services. Please review and let me know which term (if any) we should be using in this context.

Reply:
I'm indeed talking about the name, and according to your link, the relevant term for our replication feature is the Remote-copy function

Reply:
thanks, @oriyarde. I've updated accordingly.
DONE